src/main/java/net/patchworkmc/patcher/transformer/PatchworkTransformer.java

```diff
@@ -142,6 +147,20 @@ public void accept(String name, byte[] content) {
 			objectHolderClasses.add(name);
 		}
 
+		if (!capabilityInjectRewriter.getInjects().isEmpty()) {
+			accessWidenings.makeClassPublic();
+
+			for (CapabilityInject inject : capabilityInjectRewriter.getInjects()) {
```

Comment:
This doesn't seem needed except for the class access since these fields are set via generated methods